### PR TITLE
chore: verify view & exec perms for crtadmins

### PR DESCRIPTION
**Pull request by MatousJobanek**
verifies that the expected Role & RoleBindings resources are created

paired with: https://github.com/codeready-toolchain/host-operator/pull/510

### CODE REVIEW

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -300,7 +284,55 @@ func rbacEditRoleBinding() namespaceObjectsCheck {
 	}
 }
 
+func crtadminViewRoleBinding() namespaceObjectsCheck {
+	return func(t *testing.T, ns *v1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
```
Comment (xcoulon) [inline testsupport/tiers/checks.go:288]: not a blocker here, but rather a general thought: I wonder if it's better to use Go API or external YAML file to describe the expectation. WDYT?

**Review by xcoulon — APPROVED** [review]

File: testsupport/tiers/checks.go (hunk 1 above)
Comment (MatousJobanek) [reply testsupport/tiers/checks.go:288]: what do you mean by Go API?

I'm fine with using it as it is, but also open to any discussion/proposals for improvement - feel free to add it to the next sync call agenda.

Comment (MatousJobanek) [reply testsupport/tiers/checks.go:288]: as for the external file - I guess that it's better to use just go types & functions - it makes the objects/resources configurable for other tiers that just contain different numbers/values

**Pull request merged**